tests/unittests/utils/test_unique_filename_maker.py

```
@@ -111,12 +112,14 @@ def cleanup():
             id="{ignore_existing=True, idempotent=True}",
         ),
         pytest.param(
-            {"ignore_exts": True, "idempotent": False},
+           {"ignore_exts": True, "idempotent": False},
             id="{ignore_exts=True, idempotent=False}",
```

Flake8 (7.2.0)

[error] continuation line unaligned for hanging indent

(E131)